include/zephyr/drivers/spi.h

```diff
@@ -390,7 +390,7 @@ struct spi_dt_spec {
  * data from the devicetree.
  *
  * Important: multiple fields are automatically constructed by this macro
- * which must be checked before use. @ref spi_is_ready performs the required
+ * which must be checked before use. @ref spi_is_ready_dt performs the required
```

Comment:
Instead of changing this, please add something like the following:
```
@deprecated Use @c K_KERNEL_STACK_DECLARE() instead.
```

Reply:
Do you mean something like that?

<pre>
  * data from the devicetree.
  *
  * Important: multiple fields are automatically constructed by this macro
  * which must be checked before use. @ref spi_is_ready performs the required
  * @ref device_is_ready checks.
+ * @deprecated Use @ref spi_is_ready_dt instead.
  *
  * This macro is not available in C++.
  *
</pre>

Reply:
Fixed.